openshift/e2e-common.sh

```diff
@@ -135,8 +134,14 @@ function update_csv(){
     configMap:
       name: "ko-data-serving"
       items:
-        - key: "knative-serving-ci.yaml"
-          path: "knative-serving-ci.yaml"
+        - key: "1-serving-crds.yaml"
```

Comment:
We should remove the configmap approach too. Eventing just [uses](https://github.com/openshift-knative/eventing/blob/main/openshift/e2e-common.sh#L91) `make generated-files... install-eventing` which updates the files from the midstream repo here and installs eventing. Why not install via the S-O? Btw when update-to-head.sh is called on release-next-ci it will update the configs from main but also the files under /config that we need.

Reply:
Sure, we should. But do you want me to include the configmap approach change in this PR as well?

Reply:
It is ok let's do it in another I will stamp this one.